workflows/proteomics/openms-metaprosip/metaprosip-tests.yml

```
@@ -0,0 +1,39 @@
+- doc: Test outline for MetaProSIP
+  job:
+    Fasta Database:
+      class: File
+      location: https://abibuilder.cs.uni-tuebingen.de/archive/openms/galaxy-testdata/metaprosip/Zeitz_1and2_454AllContigs_HEX.fasta
+      filetype: fasta
+    Centroided LC-MS datasets:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: Zeitz_SIP_13-II_020_picked.mzML
+        location: https://abibuilder.cs.uni-tuebingen.de/archive/openms/galaxy-testdata/metaprosip/Zeitz_SIP_13-II_020_picked.mzML
+        filetype: mzml
+    Precursor monoisotopic mass tolerance (ppm): '10.0'
+    Fixed modifications: Carbamidomethyl (C)
+    Variable modifications: Oxidation (M)
+    Labeled element: C
+  outputs:
+    Peptide centric result:
+      element_tests:
+        Zeitz_SIP_13-II_020_picked.mzML:
+          asserts:
+          - that: "has_n_lines"
+            n: 85
+          - that: "has_text"
+            text: "Peptide Sequence"
+          - that: "has_text"
+            text: "AATGTPSPAGSPPPIVPAPK"  
+    Feature fitting result:
+      element_tests:
+        Zeitz_SIP_13-II_020_picked.mzML:
+          asserts:
+          - that: "has_n_lines"
+            n: 4091
+          - that: "has_text"
+            text: "Group 1"
+          - that: "has_text"
+            text: "LYSFHTLHQTYMK"  
```

```suggestion
          asserts:
          - that: has_n_lines
            n: 85
          - that: has_text
            text: Peptide Sequence
          - that: has_text
            text: AATGTPSPAGSPPPIVPAPK
    Feature fitting result:
      element_tests:
        Zeitz_SIP_13-II_020_picked.mzML:
          asserts:
          - that: has_n_lines
            n: 4091
          - that: has_text
            text: Group 1
          - that: has_text
            text: LYSFHTLHQTYMK 
```
pretty minor, doesn't have any impact and deserializes to the same dict